src/components/skeleton.js

```diff
@@ -0,0 +1,23 @@
+export const createSkeletonItem = () => {
+  const list = document.createElement("li");
+  list.classList.add("skeleton-item");
+
+  const thumbnail = document.createElement("div");
+  thumbnail.classList.add("skeleton-thumbnail");
+
+  list.appendChild(thumbnail);
+
+  return list;
+};
+
+export const createSkeleton = (count = 20) => {
+  const fragment = document.createDocumentFragment();
+  for (let i = 0; i < count; i++) {
+    fragment.appendChild(createSkeletonItem());
+  }
```

Comment:
선언형 프로그래밍을 통해서 보다 명시적으로 코드를 작성해봐도 괜찮을 것 같아요!

```suggestion
  fragment.append(...Array.from({ length: count }, createSkeletonItem));
```

축약하지 않은 2줄의 코드는 다음과 같습니다.

```js
  const skeletonItems = Array.from({ length: count }, () => createSkeletonItem());
  fragment.append(...skeletonItems);
```

Reply:
@sannimdev 
자바스크립트도 익숙지 않지만 선언형 프로그래밍을 사용해야한다는 것이 쉽게 인지가 되지 않네요. 😭 
알려주셔서 감사합니다!
32f5a4fc2b9e23babe76394d27b728fcda3fae3b